frontend/src/css/components/doc-prev.css

```diff
@@ -8,6 +10,25 @@
   columns: 2;
 }
 
+.doc-prev:hover {
+  /*
+    This is needed so it does not show white, it shows gray/black
+  */
+  border: 2px solid var(--background-color-2);
```

Comment:
Can you set the border to this by defult them but set the width to 0 then just set it to 2